pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -82,25 +83,28 @@ func newCephStatusChecker(context *clusterd.Context, clusterInfo *cephclient.Clu
 }
 
 // checkCephStatus periodically checks the health of the cluster
-func (c *cephStatusChecker) checkCephStatus(monitoringRoutines map[string]*opcontroller.ClusterHealth, daemon string) {
+func (c *cephStatusChecker) checkCephStatus(monitoringRoutines *sync.Map, daemon string) {
 	// check the status immediately before starting the loop
-	c.checkStatus(monitoringRoutines[daemon].InternalCtx)
+	v, _ := monitoringRoutines.Load(daemon)
```

Comment:
dont we need check for `ok` here or else if we are not able to load it will be nil and below code will panic?